integration/vcpus/default_vcpus_test.sh

```diff
@@ -17,21 +17,31 @@ source "${dir_path}/../../lib/common.bash"
 source "${dir_path}/../../.ci/lib.sh"
 KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
 name="${name:-default_vcpus}"
+IMAGE="${IMAGE:-docker.io/library/busybox:latest}"
```

Comment:
Maybe worth using quay.io/prometheus/busybox:latest to avoid docker pull limit
looks fine with me other than that

Reply:
@snir911 changes applied